nixos/modules/services/databases/pgmanage.nix

```diff
@@ -177,28 +176,41 @@ in {
     };
   };
 
+in {
+
+  # postage has been renamed to pgmanage. In order to get security updates we
+  # need to switch from postage to pgmanage in 17.09. However, just renaming the
+  # options from services.postage to services.pgmanage will cause evaluation
+  # failures for users which is not acceptable within a release branch. So we
+  # keep the existing services.postage:
+  options.services.postage = pgmanageOptions;
+  # And define the defaults of the new services.pgmanage in terms of services.postage:
+  options.services.pgmanage = flip mapAttrs pgmanageOptions (name: option:
+    option // { default = config.services.postage.${name}; }
```

Comment:
This will rebuild the manual every time a postage option is changed. Just setting the pgmanage options in the `config` attrset should be fine because using both interfaces shouldn't occur. To be safe, you would use `mkDefault` to set the pgmanage options.